PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,36 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
```

Comment:
staging url likely won't work with VPN?

Reply:
you mean *without*, right Jay?

Reply:
yeah, it was a long day yesterday. edited original message for clarity

Reply:
Unfortunately I still lack an understanding of whether or not this URL is acceptable...